lib/kits/core-ui/components/helpers/component.js

```diff
@@ -0,0 +1,51 @@
+const componentHelper = {
+    addBlockerListenerTo: (container, destroyCallback) => {
+        const headNav = document.querySelector('.sgn__nav');
+        const blocker = document.createElement('div');
+        blocker.className = 'sgn-blocker';
+        componentHelper.insertAfter(headNav, blocker);
+        blocker.addEventListener(
+            'click',
+            (e) => {
+                e.stopPropagation();
+
+                componentHelper.destroyBlocker(container);
+
+                if (typeof destroyCallback === 'function') {
+                    destroyCallback(e);
+                }
+            },
+            false
+        );
+    },
+    destroyBlocker: (container) => {
```

Comment:
What is "blocker"?

Reply:
Blocker is the container outside the modal that has click listener to close the modal. Maybe destroyModal is a better name 😄